packages/valory/skills/abstract_round_abci/tests/test_base_rounds.py

```diff
@@ -115,10 +116,10 @@ def test_run(
         with pytest.raises(
             TransactionNotValidError,
             match=re.escape(
-                "Expecting serialized data of chunk size 2, got: None in round_id"
+                "Expecting serialized data of chunk size 2, got: 0xZZZ in round_id"
             ),
         ):
-            test_round.check_payload(DummyTxPayload("agent_1", "value"))
+            test_round.check_payload(DummyTxPayload("agent_1", "0xZZZ"))
```

Comment:
had to be introduce this since an empty dict is no longer returned after introducing the `data` property on the `DummyPayload`